### ISSUE TITLE
tests/benchmarks/timing_info/testcase.yaml#test failed @ nrf52_pca10040

### ISSUE THREAD

**Issue by inakypg** [zephyrproject-rtos/zephyr#1218]
```
starting test - Time Measurement
Timing Results: Clock Frequency: 0 MHz
Context switch                             ***** MPU FAULT *****
  Executing thread ID (thread): 0x2000110c
  Faulting instruction address:  0x200024a8
  Instruction Access Violation
Fatal fault in ISR! Spinning...
```

gitrev 4e439653aaaf0c2ab8a4333cebb1e1385905354a

**Comment by nashif** [comment]
@carlescufi  FYI

**Comment by youvedeep** [comment]
Further there is another thread at GitHub where discussion about this is tracked.

**Comment by youvedeep** [comment]
https://github.com/zephyrproject-rtos/zephyr/pull/1095

**Comment by carlescufi** [comment]
@nashif I am following up in the #1095 thread, thanks!

**Comment by AdithyaBaglody** [comment]
I feel the issue is with the AAPCS for ARM. if the scratch registers are saved it should be okay.

**Comment by nashif** [comment]
fixed by #1095
